### PR TITLE
FA-33: Added 'Vary' to the activity page for shooting-clubs, new-dealer and museums.

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
**Changes**

            * Added 'Vary' selection to shooting-clubs, new-dealer and museums index.js pages.
            * Updated fields.json to include new option.
            * Updated submission.js in shooting-clubs, new-dealer and museums logic for new option.
            * Updated index.js in shooting-clubs, new-dealer and museums for new fork step which follows the same as 'renew'.
            * Added additional unit tests for 'vary' in base.test.js
            * Changed find in submission.js for shooting-clubs, new-dealer and museums

### CODE REVIEW

File: apps/common/controllers/base.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ module.exports = class BaseController extends Controller {
 
   locals(req, res) {
     const locals = super.locals(req, res);
-    const renew = req.sessionModel.get('activity') === 'renew' && locals.renew === true;
+    const activity = req.sessionModel.get('activity');
+    const renew = (activity === 'renew' || activity === 'vary') && locals.renew === true;
```
Comment (Alex-Swann) [inline apps/common/controllers/base.js:10]: remove `=== true`

File: package.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
     "type": "git",
     "url": "https://github.com/UKHomeOffice/firearms"
   },
+  "engines": {
+    "node": "14.16.1"
+  },
```
Comment (Alex-Swann) [inline package.json:13]: remove

**Review by Alex-Swann — APPROVED** [review]

File: apps/common/controllers/base.js (hunk 1 above)
Comment (SarahJaneLuff) [reply apps/common/controllers/base.js]: removed

File: package.json (hunk 2 above)
Comment (SarahJaneLuff) [reply package.json]: Removed Node Engines

**Pull request merged**